(2026.5.2) Docker を使用している場合に 2026.5.1 以前の環境からのアップグレードに失敗する

### 💡 Summary

ERR_PNPM_ABORTED_REMOVE_MODULES_DIR_NO_TTY で pnpm install に失敗するらしい
pnpm v11に上げたのがあるかも

workaroundとしては `pnpm_config_confirm_modules_purge=false` を環境変数に与える（もしくは `pnpm install --config.confirmModulesPurge=false` を実行する）と成功する可能性がある？

### 🥰 Expected Behavior

成功する

### 🤬 Actual Behavior

失敗する

### 📝 Steps to Reproduce

N/A

### 💻 Frontend Environment

```markdown
N/A
```

### 🛰 Backend Environment (for server admin)

```markdown
* Installation Method or Hosting Service: Docker
* Misskey: 2026.5.2
```

### Do you want to address this bug yourself?

- [ ] Yes, I will patch the bug myself and send a pull request

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -3,12 +3,6 @@
 ### General
 - Fix: Dockerで起動に失敗する問題を修正
 
-### Client
--
-
-### Server
--
-
 
 ## 2026.5.2
 

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,7 +1,7 @@
 ## Unreleased
 
 ### General
--
+- Fix: Dockerで起動に失敗する問題を修正
 
 ### Client
 -

--- a/Dockerfile
+++ b/Dockerfile
@@ -74,6 +74,8 @@ FROM --platform=$TARGETPLATFORM node:${NODE_VERSION}-slim AS runner
 ARG UID="991"
 ARG GID="991"
 
+ENV PNPM_CONFIG_VERIFY_DEPS_BEFORE_RUN=false
+
 RUN apt-get update \
 	&& apt-get install -y --no-install-recommends \
 	ffmpeg tini curl libjemalloc-dev libjemalloc2 \

